tests/scripts/test_pip_audit_gate.py

```
@@ -0,0 +1,158 @@
+from pathlib import Path
+from textwrap import dedent
+
+import pytest
+
+# Load scripts/pip_audit_gate.py by path — it isn't a package, so we can't import it
+# via `from scripts.pip_audit_gate import ...` without adjusting sys.path. Loading
+# explicitly keeps the test self-contained.
+_REPO_ROOT = Path(__file__).resolve().parents[2]
+_MODULE_PATH = _REPO_ROOT / "scripts" / "pip_audit_gate.py"
+_spec = importlib.util.spec_from_file_location("pip_audit_gate", _MODULE_PATH)
+assert _spec is not None and _spec.loader is not None
```

Ruff (0.15.10)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)